packages/react/src/components/Modal/tokens.ts

```diff
@@ -0,0 +1,19 @@
+import { DefaultTokens } from "@openfun/cunningham-tokens";
+
+export const tokens = (defaults: DefaultTokens) => {
+  return {
+    "background-color": defaults.theme.colors["greyscale-000"],
+    "border-radius": "4px",
+    "border-color": defaults.theme.colors["greyscale-300"],
+    "box-shadow": "0px 1px 2px 0px #0C1A2B4D",
+    color: defaults.theme.colors["greyscale-900"],
+    "backdrop-color": "#0C1A2B99",
+    "title-font-weight": defaults.theme.font.weights.bold,
+    "content-font-size": defaults.theme.font.sizes.m,
+    "content-font-weight": defaults.theme.font.weights.regular,
+    "content-color": defaults.theme.colors["greyscale-800"],
+    "small-width": "300px",
+    "medium-width": "600px",
+    "large-width": "75%",
```

Comment:
To improve DX, it's better to start the token name by the property then the variant.
In this way, by typing `wid`, autocompletion will show all tokens related to `width` property.